projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-row-dimension-content.component.html

```diff
@@ -13,16 +13,15 @@
 
 <ng-template #headerTemplate let-column>
     <div class='igx-grid__tr--header igx-grid__row-indentation--level-{{getLevel()}}'>
-        <igx-icon [attr.draggable]=" false" (click)="toggleRowDimension($event)">
+        <igx-icon [attr.draggable]="false" (click)="toggleRowDimension($event)">
```

Comment:
I assume here [attr.draggable] can also be removed.

Reply:
We've decided to not remove the `attr.draggable` from the grid yet, to be sure that the behavior remains the same.